src/pybind/mgr/cephadm/service_discovery.py

```diff
@@ -113,6 +114,7 @@ def index(self) -> str:
 <p><a href='prometheus/sd-config?service=node-exporter'>Node exporter http sd-config</a></p>
 <p><a href='prometheus/sd-config?service=haproxy'>HAProxy http sd-config</a></p>
 <p><a href='prometheus/rules'>Prometheus rules</a></p>
+<p><a href='prometheus/sd-config?service=ceph-exporter'>Ceph exporter http sd-config</a></p>
```

Comment:
nit: I think it's better to move this line up just to group all the sd-config in the index page.